homeassistant/components/media_player/firetv.py

```diff
@@ -93,9 +105,34 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     name = config[CONF_NAME]
     get_sources = config[CONF_GET_SOURCES]
 
-    device = FireTVDevice(ftv, name, get_sources)
-    add_entities([device])
-    _LOGGER.debug("Setup Fire TV at %s%s", host, adb_log)
+    if host in hass.data[DATA_KEY]:
+        _LOGGER.warning("Platform already setup on %s, skipping.", host)
```

Comment:
We don't end logging messages with period.

Reply:
Fixed in https://github.com/home-assistant/home-assistant/pull/21419/commits/546d1d1b05b7f0bf1c9e9ff0490ccc374a6f2336